### ISSUE TITLE
Views exposed form fields: Improve AT usability with aria-describedby

### ISSUE THREAD

**Issue by alisonjo315** [CU-CommunityApps/cwd_base#65]
Some form fields have description/help text under them, in addition to field labels.

These fields would be more accessible/assistive-tech-user-friendly if the description text were programmatically associated with its field, via `aria-describedby`.

Example implementation, based on text search/filter on [Einaudi > News](https://einaudi.cornell.edu/discover/news) (edited for length):
```html
<div class="js-form-item form-item js-form-type-textfield form-item-text-search js-form-item-text-search">
  <label for="edit-text-search--2">Search</label>
  <input type="text" id="edit-text-search--2" name="text_search" value="" aria-describedby="edit-text-search--2--description">
  <div id="edit-text-search--2--description" class="description">
    Simple text search.
  </div>
</div>
```

----

### To do
- [ ] Decide: We should do this, right?
- [ ] Decide: Just target views exposed forms, right?
    * Only other public-facing fields I can think of are webforms, and we don't have Webform included with our base Drupal implementation.  (Site search, yes, but the site search text field doesn't have help text.)
- [ ] Create template override (`core/modules/system/templates/input.html.twig`)
    * May need to add template suggestion to cwd_base.theme.
- [ ] Test / review / release
- [ ] Optional: Create companion issue on cwd_claro for content entry fields.  (Why not make those forms more usable, right?)

----

_Suggested by Hank on Slack, with this reference: https://www.tpgi.com/using-aria-describedby-to-provide-helpful-form-hints/_

**Comment by sg667** [comment]
Yes, this should be done - I believe we have even done this already on some of our sites, potentially not all though as not all sites have form fields with additional descriptive text. (Although I may just be thinking of Laravel stuff, not Drupal)

While I'm not the most knowledgeable on the team about form creation on Drupal, is this going to add an optional field to add descriptive text when creating forms?

**Comment by alisonjo315** [comment]
Wrinkle: There are no views on our Drupal demo site that have exposed forms, so we can't actually show/test it on our demo site! -- maybe let's add an exposed filter to a view on our demo site?? https://github.com/CU-CommunityApps/cd_drupal_upstream/issues/80
